src/BenchmarkDotNet/ConsoleArguments/ConfigParser.cs

```diff
@@ -134,7 +134,15 @@ private static bool Validate(CommandLineOptions options, ILogger logger)
             foreach (var coreRunPath in options.CoreRunPaths)
                 if (coreRunPath.IsNotNullButDoesNotExist())
                 {
-                    logger.WriteLineError($"The provided path to CoreRun: \"{coreRunPath}\" does NOT exist. Please remember that BDN expects path to CoreRun.exe (corerun on Unix), not to Core_Root folder.");
+                    if (Directory.Exists(coreRunPath.FullName))
+                    {
+                        logger.WriteLineError($"The provided path to CoreRun: \"{coreRunPath}\" exists but it's a directory, not an executable. You need to provide path to CoreRun.exe (corerun on Unix), not to the Core_Root folder.");

```

Comment:
perhaps instead of 
`You need to provide path to CoreRun.exe (corerun on Unix), not to the Core_Root folder.` 
you could say something like  
`You need to include CoreRun.exe in the path` 
? which is really the culprit here. 

`path` is ambiguous - to some people the path to a.exe means c:\dir_a.exe_is_in\, to others it means c:\dir_a.exe_is_in\a.exe.

Reply:
> path is ambiguous - to some people the path to a.exe means c:\dir_a.exe_is_in, to others it means c:\dir_a.exe_is_in\a.exe.

I did not know about that, for me it was always `c:\dir_a.exe_is_in\a.exe`. Since my understanding of path is the source of the confusion I am going to apply your suggestion.